lib/test_module_C.dx

```diff
@@ -0,0 +1,6 @@
+
+
+test_module_C_val = 23
+
+interface FooClass a
+  fooMethod : a -> a
```

Comment:
That doesn't quite work because it wouldn't be able to read the prelude. What we really need is a better way to customize a list of search paths rather than a single directory. DEXPATH environment variable? But that feels like a change for another PR.

Reply:
Did you remove `--prelude-file`? I thought that we could still point to the right prelude file, but still end up being able to use a different lib path?

Reply:
Oh good point. `--prelude-file` should do the trick.